seqr/views/apis/anvil_workspace_api.py

```diff
@@ -155,11 +154,19 @@ def create_project_from_workspace(request, namespace, name):
         project, individual_records=pedigree_records, user=request.user
     )
 
-    # Send an email to all seqr data managers
+    # Upload sample IDs to a file on Google Storage
+    ids_path = _get_loading_project_path(project, request_json['sampleType']) + '/{guid}_ids.txt'.format(guid=project.guid)
     try:
-        _send_load_data_email(project, updated_individuals, data_path, request_json['sampleType'], request.user)
+        save_data_to_gs(ids_path, '\n'.join(['s'] + [individual.individual_id for individual in updated_individuals]),
+                        user=request.user)
```

Comment:
I found that the ID file is not possible to be pre-existing.  It is because that the new project GUID in the ID file path must be different from that of the existing projects. Therefore, I don't rename any existing ID file with a sequence number.